### ISSUE TITLE
map from pixel coords to text position

### ISSUE THREAD

**Issue by 3b** [3b/3b-bmfont#3]
need some way to map from pixel coordinates back to position in string, for mouse interaction etc.

**Comment by Shinmera** [comment]
I don't think this should be the job of the glyph renderer, but rather of the character layouting engine.
